docs/spec/blockchain/blockchain.md

```diff
@@ -187,6 +187,8 @@ type DuplicateVoteEvidence struct {
 }
 ```
 
+Votes are lexicographically sorted on `BlockID`.
```

Comment:
this should be added to the spec repo. the specs in this repo are going to be removed so we dont have duplicates

Reply:
added to the spec repo